src/core/server/http/http_config.ts

```diff
@@ -73,6 +73,11 @@ export const config = {
       host: schema.string({
         defaultValue: 'localhost',
         hostname: true,
+        validate(value) {
+          if (value === '0') {
```

Comment:
Should we add a deprecation warning for it https://github.com/elastic/kibana/blob/master/src/core/server/config/deprecation/core_deprecations.ts ? as I understand you aren't going to backport the PR to 7.x, are you?

Reply:
No you're right, this isn't going to be backported. But the deprecation warning should go into `7.x` right? If so, I'll do that as a separate PR directly to `7.x`.

Reply:
yes, a separate PR is file. I just asked here to avoid creating an issue

Reply:
PR: #87471